code/datums/trading/ai.dm

```diff
@@ -27,24 +27,66 @@ They sell generic supplies and ask for generic supplies.
 	)
 	possible_wanted_items = list(
 		/obj/item/device/                        = TRADER_SUBTYPES_ONLY,
+		/obj/item/device/kit					 = TRADER_BLACKLIST_ALL, // They're impossible to make or get outside of mining, just annoys traders.
 		/obj/item/device/assembly                = TRADER_BLACKLIST_ALL,
 		/obj/item/device/assembly_holder         = TRADER_BLACKLIST_ALL,
-		/obj/item/device/encryptionkey/syndicate = TRADER_BLACKLIST,
+		/obj/item/device/encryptionkey			 = TRADER_BLACKLIST_ALL, //Why does ai want NT encryption keys?
```

Comment:
Can be acquired easily enough

Reply:
Acquired? Maybe. But it doesn't make sense. Why should an ai trader want NanoTrasen encryption keys? Is the trading beacon selling NanoTrasen communication encryption keys [and thus how to decrypt NT communications] to someone..? Why should it want those? Especially outrageous are when it wants sth like security/command encryption keys - security is not going to let joe merch listen in on their communications, and neither will command.

Do you still want this change despite this reasoning?

Reply:
As a sidenote, these basically cannot be acquired without antag activity as unused ones are only located in their respective departments' lockers, and used ones are not going to be handed out /willingly/

Reply:
Well, you could just blacklist the subtypes.
Default encryption keys are easy to acquire.

Reply:
That seems acceptable - will be done.